javalib/src/main/scala/java/util/ArrayList.scala

```diff
@@ -44,50 +71,134 @@ class ArrayList[E] private (private[ArrayList] val inner: js.Array[E])
   }
 
   def size(): Int =
-    inner.length
+    if (targetPureWasm) {
+      _size
+    } else {
+      innerJS.length
+    }
```

Comment:
```suggestion
    if (targetPureWasm)  _size
    else innerJS.length
```
so that it's a "two-liner if/else". Otherwise the whole body needs braces, as per the coding style.